src/framework/dropdown/DropdownGroup.jsx

```diff
@@ -0,0 +1,41 @@
+
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+import Dropdown from './Dropdown.jsx';
+
+const DropdownGroup = (props) => {
+  const optionToList = props.options.reduce(
```

Comment:
Comment from initial review:

> The result is the list itself (like `options` or `optionList`), but optionToList indicates a method rather than a list I think.